sgv2-restapi/src/main/java/io/stargate/sgv2/restsvc/resources/schemas/Sgv2KeyspacesResource.java

```diff
@@ -0,0 +1,246 @@
+/*
```

Comment:
How much of this is new and how much is copy-paste? This seems like something we can do in place in `io.stargate.web.restapi.resources.v2.schemas.KeyspacesResource`

Reply:
If you want to keep tests green then just change swap out the DataStore for gRPC on a case by case basis rather than running two services in parallel.

Reply:
I have been going back and forth wrt trying to mix-and-match old and new types; both approaches have (IMO) benefits. Copy-pasting -- for now, not necessarily in the end -- has the benefit of using optimal approach, minimizing dependencies. To me attempting to mix-and-match so far has seemed to pull in unwanted dependencies and significantly complicate progress.

But we can definitely discuss this further; this is very much WIP.

Reply:
And as to copy-paste: only things that can be used without modifications, and that do not bring other dependencies, are reused.

Resource classes in particular cannot be reused: some value types conceivably could.

But as to resources: this is what I was trying to suggest earlier -- signatures, annotations are potentially reusable, and could be extracted into interfaces (or abstract classes), implemented by v1 and v2 separately.
This is possible. But it is not clear it'd be worth the complexity for reader caused by interface/impl division of every Resource class.